includes/gateways/stripe/includes/admin/admin-actions.php

```diff
@@ -282,6 +346,12 @@ function give_stripe_show_connect_banner() {
 		$status = false;
 	}
 
+	// Additional Check: For multiple accounts management.
+	$all_accounts = give_stripe_get_all_accounts();
+	if ( is_array( $all_accounts ) && count( $all_accounts ) > 0 ) {
```

Comment:
@mehul0810 does not need to use `is_array` and `count`. 
```suggestion
	if ( $all_accounts ) {
```
Update `give_stripe_get_all_accounts` function  to return result only in `array` format.